linera-service/tests/end_to_end_tests.rs

```diff
@@ -40,6 +40,46 @@ fn get_fungible_account_owner(client: &ClientWrapper) -> AccountOwner {
     AccountOwner::User(owner)
 }
 
+#[cfg(feture = "ethereum")]
+struct EthereumTrackerApp(ApplicationWrapper<ethereum_tracker::EthereumTrackerAbi>);
+
+#[cfg(feture = "ethereum")]
```

Comment:
```suggestion
#[cfg(feature = "ethereum")]
struct EthereumTrackerApp(ApplicationWrapper<ethereum_tracker::EthereumTrackerAbi>);

#[cfg(feature = "ethereum")]
```

(Did this compile? `cargo check-all-features --all-targets` should fail, shouldn't it?)

Reply:
Yes, strangely no compiler issues arose. I addressed that.